code/components/devtools-five/src/TimecycleEditor.cpp

```diff
@@ -2,7 +2,6 @@
 #include <CoreConsole.h>
 #include <ConsoleHost.h>
 #include <GameInit.h>
-#include <HostSharedData.h>
```

Comment:
Might this be related to the build failure in AppVeyor?

```
C:\projects\fivem\code\components\devtools-five\src\TimecycleEditor.cpp(156,7): error C2653: 'launch': is not a class or namespace name [C:\save\build\client\master\five\devtools-five.vcxproj]
C:\projects\fivem\code\components\devtools-five\src\TimecycleEditor.cpp(156,15): error C3861: 'IsSDK': identifier not found [C:\save\build\client\master\five\devtools-five.vcxproj]
C:\projects\fivem\code\components\devtools-five\src\TimecycleEditor.cpp(156,27): error C2653: 'launch': is not a class or namespace name [C:\save\build\client\master\five\devtools-five.vcxproj]
C:\projects\fivem\code\components\devtools-five\src\TimecycleEditor.cpp(156,35): error C3861: 'IsSDKGuest': identifier not found [C:\save\build\client\master\five\devtools-five.vcxproj]
```

Reply:
Compiles locally fine, but not sure why I removed this include. Reverted this specific change and force pushed.